crates/pessimistic-proof/src/local_exit_tree/tests.rs

```diff
@@ -25,6 +26,32 @@ fn test_local_exit_tree_basic() {
     );
 }
 
+fn local_exit_tree_is_subtree(leaf_count_a: usize, leaf_count_b: usize) {
+    assert!(leaf_count_a <= leaf_count_b);
+    let mut let_a = LocalExitTreeData::<Keccak256Hasher>::new();
+    for _ in 0..leaf_count_a {
+        let_a.add_leaf(random());
+    }
+    let mut let_b = let_a.clone();
+    for _ in 0..leaf_count_b - leaf_count_a {
+        let_b.add_leaf(random());
+    }
+
+    let let_a_frontier = LocalExitTree::from(&let_a);
+    let proof = let_b.get_proof(leaf_count_a as u32);
+    assert!(let_a_frontier.is_subtree(let_b.get_root(), let_b.get(0, leaf_count_a), proof));
+}
+
+#[test]
+fn test_local_exit_tree_is_subtree_different_sizes() {
+    local_exit_tree_is_subtree(100, 200);
+}
+
+#[test]
+fn test_local_exit_tree_is_subtree_empty() {
+    local_exit_tree_is_subtree(0, 100);
+}
```

Comment:
Maybe adding at least one failing case? Also the case with two trees that are the same?

Reply:
Done 👍